deployments/icesat2/image/binder/environment.yml

```diff
@@ -16,18 +16,19 @@ dependencies:
   - nomkl
   - s3fs=0.2
   - gcsfs=0.2
-  - intake=0.4
+  - intake=0.5
```

Comment:
Do we need to specify `intake` as a dependency? I think that having `intake-xarray` takes care of this